### ISSUE TITLE
rake ow:ssh[env,instance] works, but not ow:console

### ISSUE THREAD

**Issue by dblock** [artsy/momentum#15]
```
~/source/gemini/dblock (master)$ rake ow:console[production,sidekiq1]
Starting remote console... (use Ctrl-D to exit cleanly)
ssh -t -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -i ...

`resolve_symbol_connection': 'sidekiq1' database is not configured. Available: ["development", "production"] (ActiveRecord::AdapterNotSpecified)
```

**Comment by joeyAghion** [comment]
This is by design rather than a bug. (Many instances _can't_ start a rails console.)

Would customizing the `rails_console_layer` [config](https://github.com/artsy/momentum#configuration) to be `'sidekiq'` solve your particular problem?

**Comment by dblock** [comment]
Yes in this specific case, but I don't *always* want a sidekiq instance, only when I want to do image processing tasks. Why wouldn't any of these tasks (ssh, console, foobar) just support the `,host` syntax?

**Comment by joeyAghion** [comment]
I'd be OK with a modification that allows you to specify an instance, but I'd also like there to remain a default that (1) selects a likely rails-compatible layer and (2) spreads potential consoles out among instances of that layer.

We don't have conventions for this yet, but you could borrow from the `ow:ssh` task that accepts a layer _or_ instance name as the 2nd argument.
